plugins/modules/jcliff.py

```diff
@@ -772,6 +772,21 @@
           - Create messaging activemq.
         type: dict
         suboptions:
+          server_property:
+            description:
+              - Creates and sets messaging-activemq server properties.
+              type: list
```

Comment:
Looking at the sanity test CI failing I believe you have an indent problem here (type and suboptions  same-level with description?)

Reply:
I have now fixed the indentation. 
I also noticed I hadn't set elements: dict on server_property.